index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const { getNostalgia, getCategory, getSlug, getDecade } = require('./controllers/nostalgia')
+
+const app = express()
+
+app.get('/', getNostalgia)
```

Comment:
Wasn't expecting that from the root route. A user is gonna just go to the base URL, that should present the docs.

![image](https://user-images.githubusercontent.com/5775716/80812387-db120280-8b95-11ea-85e1-be72fb3a1b52.png)